compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -456,7 +456,6 @@ OMR::ARM64::TreeEvaluator::sloadEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return commonLoadEvaluator(node, TR::InstOpCode::ldrshimmx, 2, cg);
    }
 
-// also handles cloadi
 TR::Register *
 OMR::ARM64::TreeEvaluator::cloadEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Does this need to be renamed - cload is also gone based on the other removes no?

Reply:
Yes, there is no more creations of `cload` IL Opcode. But `cloadEvaluator()` will be removed in future PR.